### ISSUE TITLE
Crash with Lycanites

### ISSUE THREAD

**Issue by Sereath** [Elite-Modding-Team/MoCreaturesExtended#158]
https://gist.github.com/Sereath/f5a8b0aee7acbc26871181e36f504c1d

Really only happened after about the fifth new world I create while developing this modpack.
Crashed after a few seconds.

And I don't think Lycanites is in development at the moment.

Mo' Creatures Extended 12.4.3
Lycanites Mobs 2.0.8.9

**Comment by IcarussOne** [comment]
Possibly related to #134, as it's also a similar crash.

**Comment by Sereath** [comment]
That's so odd that it might be a combination of more than just these two mods.
Thank you for the info, I'll see what I can do for now.

**Comment by Sereath** [comment]
Yeah, it is as they said, I tried spawning a Fishy mob inside an acid pool this time and got the same crash:

https://gist.github.com/Sereath/af279bb4d9197ec3f355b32d76c5ef94

Edit:
Same with a dolphin, safe to say it's every aquatic creature.
https://gist.github.com/Sereath/26833c82cf251571df76799a1210f5d1

**Comment by IdkWhatNameThingyToPut** [comment]
Ran into this problem tonight here is the crash report:
https://mclo.gs/hMgblAL

I read the other post linked here and I'm guessing a workaround until a proper fix is out is to disable structures from the lycanites mod which I might do for my modpack anyway if I don't I'll probably make it so that structures only spawn in specific dimensions where mobs from mo creatures extended don't spawn anyway.
